microsoft-edge/web-platform/local-network-access.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Controlling Local Network Access in Microsoft Edge
+description: Local Network Access in Microsoft Edge enables users to grant or deny sites' access to your local network.  # key words before col 158
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.date: 08/18/2025
+---
+# Controlling Local Network Access in Microsoft Edge
+<!-- todo: improve "description:" above, after clarify intro -->
+
+Use the Local Network Access feature to grant or deny sites' access to your<!-- todo: who is "your"?  the (end-)user's? --> local network.
```

Comment:
who is "your"? the end-user?
maybe begin article:
As an end-user, you can use the Local Network Access feature to grant or deny a website access to your local network.  This requires your admin to set policies, below.